laalaa/apps/advisers/tasks.py

```diff
@@ -1,51 +1,44 @@
 # -*- coding: utf-8 -*-
 import csv
 import logging
-import os
+import time
 import tempfile
-from threading import Thread, Event
-from time import sleep
-
+import itertools
+from celery.task import TaskSet
+import re
+from django.utils.text import slugify
+import os
+from celery import Task
 from django.conf import settings
 from django.contrib.gis.geos import Point
-from django.db import connection, transaction
-from django.utils.text import slugify
-import xlrd
+from django.core.cache import cache
+from django.db import connection
 
 from . import models
 from . import geocoder
+import xlrd
```

Comment:
From PEP8:

    Imports should be grouped in the following order:

      1. standard library imports
      2. related third party imports
      3. local application/library specific imports

    You should put a blank line between each group of imports.